wagtail_guide/tests/settings.py

```diff
@@ -79,3 +79,5 @@
 }
 
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
+
+STATIC_URL = "/static/"
```

Comment:
I'm only wondering why this was added?

Reply:
Hi @nickmoreton ,
So this error can be avoided:
https://github.com/torchbox-forks/wagtailguide/actions/runs/6809761204/job/18516669914#step:5:76